### PR TITLE
Skeleton for ex-4

### LINKED ISSUE #21: Create a Scene class

### LINKED ISSUE #26: Create a Light interface and an AmbientLight class in a lighting package
- [ ] Depends on Colour #25

### PR DESCRIPTION
**Pull request by abrahammurciano**
close #26
close #21 
partially did #23

### CODE REVIEW

File: src/main/lighting/AmbientLight.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package lighting;
+
+import primitives.Colour;
+
+/**
+ * This light models the small ammounts of light which reflected so many times that their source is indeterminable. All
```
Comment (elevin72) [inline src/main/lighting/AmbientLight.java:6]: *amount. And that's not really what ambient light is.

File: src/main/lighting/AmbientLight.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+ * geometry intersections will be affected by at least this light.
+ *
+ * @author Abraham Murciano
+ * @author Eli Levin
+ */
+public class AmbientLight {
+
+	/** The colour of the ambient light. */
```
Comment (elevin72) [inline src/main/lighting/AmbientLight.java:14]: color*

Comment (abrahammurciano) [reply src/main/lighting/AmbientLight.java:14]: https://tenor.com/view/english-do-you-speak-it-motherfucker-jules-pulp-gif-17700680

File: src/main/lighting/AmbientLight.java (hunk 1 above)
Comment (abrahammurciano) [reply src/main/lighting/AmbientLight.java:6]: That's where I got it from. If you prefer another explanation, feel free to change it.

File: src/main/lighting/AmbientLight.java (hunk 2 above)
Comment (abrahammurciano) [reply src/main/lighting/AmbientLight.java:14]: Also, see #25

**Pull request merged**